tekton/resources/checks/check-pr-has-kind-label.yaml

```diff
@@ -0,0 +1,220 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: check-labels-trigger
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["tekton.dev"]
+  resources: ["eventlisteners", "triggerbindings", "triggertemplates"]
+  verbs: ["get", "list"]
+- apiGroups: ["tekton.dev"]
+  resources: ["pipelineruns", "pipelineresources", "taskruns"]
+  verbs: ["create"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: gh-pr-label-checker
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: gh-pr-label-checker-rolebinding
+subjects:
+- apiGroup: ""
+  kind: ServiceAccount
+  name: gh-pr-label-checker
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: check-labels-trigger
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: trigger-to-check-pr-labels
+spec:
+  params:
+  - name: labels
+    value: $(body.pull_request.labels)
+  - name: gitRepository
+    value: $(body.repository.html_url)
+  - name: pullRequestUrl
+    value: $(body.pull_request.html_url)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: github-pull-request-label-checker
+spec:
+  serviceAccountName: gh-pr-label-checker
+  triggers:
+    - name: trigger-check-pr-labels
+      interceptors:
+        - cel:
+            filter: >
+              header.match('X-GitHub-Event', 'pull_request') && 
+              (body.action == 'opened' || 
+              ((body.action == 'labeled' || body.action == 'unlabeled') 
+              && body.label.name.contains('kind')))
+      bindings:
+        - name: trigger-to-check-pr-labels
+      template:
+        name: check-pull-request-labels
+---
+apiVersion: tekton.dev/v1alpha1
```

Comment:
the api group changes to `triggers.tekton.dev` in triggers v0.4.0 - which we now run - could you update all trigger resources accordingly?